lib/temporal/workflow/handle.rb

```diff
@@ -0,0 +1,55 @@
+module Temporal
+  class Workflow
+    class Handle
```

Comment:
I think this should be with the client. I think we should reserve the "workflow" area for guaranteed-deterministic stuff you'd use from inside workflow code. This is how all of our other SDKs are organized.

Reply:
I guess that makes sense, but there's going to be a handle to the Child Workflow as well (if I remember correctly)

Reply:
+1